djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt

```diff
@@ -377,6 +377,15 @@ class AnalysisConfiguration private constructor(
                 descriptor = "()V",
                 genericsDetails = ""
             )
+        ).mapByClassName() + listOf(
+            // This method will be deleted.
+            Member(
+                access = ACC_STATIC,
+                className = sandboxed(SecurityManager::class.java),
+                memberName = "<clinit>",
+                descriptor = "()V",
+                genericsDetails = ""
+            )
```

Comment:
It would be nice if there was a way to specify the bytecode modifcations that go into the sandbox that is structured more like actual code; a long list of blocks calling a bytecode look-alike DSL becomes very hard to follow very rapidly.

Reply:
Yes, this feature is growing beyond it's original intention.